packages/wallet-service/src/api/healthcheck.ts

```diff
@@ -0,0 +1,143 @@
+import middy from "@middy/core";
+import {
+    Healthcheck,
+    HealthcheckInternalComponent,
+    HealthcheckDatastoreComponent,
+    HealthcheckHTTPComponent,
+    HealthcheckCallbackResponse,
+    HealthcheckStatus,
+} from "@hathor/healthcheck-lib";
+import { getLatestHeight } from "@src/db";
+import fullnode from "@src/fullnode";
+import { closeDbConnection, getDbConnection } from "@src/utils";
+import { APIGatewayProxyHandler } from "aws-lambda";
+import { getRedisClient, ping } from "@src/redis";
+
+const mysql = getDbConnection();
+
+const checkDatabaseHeight: HealthcheckCallbackResponse = async () => {
+    try {
+        const [currentHeight, fullnodeStatus] = await Promise.all([
+            getLatestHeight(mysql),
+            fullnode.getStatus()
+        ]);
+
+        const currentFullnodeHeight = fullnodeStatus["dag"]["best_block"]["height"];
+
+        if (currentHeight === currentFullnodeHeight) {
```

Comment:
Maybe we should have a threshold here, we might still be syncing

Reply:
Done in https://github.com/HathorNetwork/hathor-wallet-service-sync_daemon/pull/110/commits/8fb42a40c15416a0c369af8e99286b5f303314c7